distributed/http/routing.py

```diff
@@ -7,7 +7,9 @@
 from tornado import web
 
 
-def _descend_routes(router, routers=set(), out=set()):
+def _descend_routes(router, routers=None, out=None):
```

Comment:
`test_sitemap` fails because of this change, so it seems like we're relying on an anti-pattern.

Reply:
Never mind, this relied on being able to pass in an empty set.